common/src/main/java/org/tango/transport/TangoMessage.java

```diff
@@ -0,0 +1,35 @@
+package org.tango.transport;
+
+import fr.esrf.TangoDs.TangoConst;
+
+/**
+ * @author Igor Khokhriakov <igor.khokhriakov@hzg.de>
+ * @since 19.02.2020
+ */
+public class TangoMessage<T> {
+    public String action;
+    public String device;
+    public String target;
+    public int dataType;
+    public T value;
```

Comment:
No serialization strategy here. I really do not like using `toString` for something other than debug. There should be either additional decoder/encoder strategy or limits on `T`.